cmd/tree-walk_test.go

```diff
@@ -29,6 +29,15 @@ import (
 // Fixed volume name that could be used across tests
 const volume = "testvolume"
 
+// list of all errors that can be ignored in tree walk operation.
+var walkResultIgnoredErrs = []error{
```

Comment:
your earlier suggestion : `var fsTreeWalkIgnoredErrs = ` is more readable i.e fsTreeWalkIgnoredErrs and xlTreeWalkIgnoredErrs